internal/cli/atlas/security/ldap/save.go

```diff
@@ -64,6 +67,7 @@ func (opts *SaveOpts) newLDAPConfiguration() *atlas.LDAPConfiguration {
 			AuthorizationEnabled:  opts.authorizationEnabled,
 			Hostname:              opts.hostname,
 			Port:                  opts.port,
+			UserToDNMapping:       []*atlas.UserToDNMapping{{Match: opts.mappingMatch, LDAPQuery: opts.mappingLdapQuery, Substitution: opts.mappingSubstitution}},
```

Comment:
Are you sure that we are not setting `UserToDNMapping` when `opts.mappingMatch`, `opts.mappingLdapQuery` and `opts.mappingSubstitution` are empty?

Our LDAP e2e test is broken https://evergreen.mongodb.com/task/mongocli_master_e2e_atlas_clusters_atlas_ldap_e2e_patch_3dc498b12ecd12496d2dd71d8e152a5f411680da_60a2ad5d1e2d170ad5a81575_21_05_17_17_52_30

Reply:
In particular this error, https://evergreen.mongodb.com/test_log/mongocli_master_e2e_atlas_clusters_atlas_ldap_e2e_patch_3dc498b12ecd12496d2dd71d8e152a5f411680da_60a2ad5d1e2d170ad5a81575_21_05_17_17_52_30/0/github.com_mongodb_mongocli_e2e_atlas.TestLDAP_Save

Reply:
Ah, thanks! I believe I fixed this. I tested different combinations with the fix:
1) Passing no new flags (works - didn't work before)
2) Passing all new flags (local validation fails)
3) MappingMatch with neither query nor substitution (local validation fails)
4) MappingMatch with Query (validates fine and saves)
5) MappingMatch with Substitution (validates fine and saves)